banderwagon/element_test.go

```diff
@@ -183,34 +179,34 @@ func TestBatchElementsToBytes(t *testing.T) {
 	got_serialised_b := serialised_points[1]
 	if expected_serialised_a != got_serialised_a {
 		panic("expected serialised point of A is incorrect ")
-
 	}
 	if expected_serialised_b != got_serialised_b {
 		panic("expected serialised point of B is incorrect ")
 	}
-
 }
 
 func TestMultiMapToBaseField(t *testing.T) {
-
 	var A, B Element
 
 	A.Add(&Generator, &Generator)
 	B.Double(&Generator)
 	B.Double(&B)
 
-	expected_a := A.MapToScalarField()
-	expected_b := B.MapToScalarField()
+	var expected_a, expected_b fr.Element
+	A.MapToScalarField(&expected_a)
+	B.MapToScalarField(&expected_b)
 
-	scalars := MultiMapToScalarField([]*Element{&A, &B})
+	var ARes, BRes fr.Element
+	scalars := []*fr.Element{&ARes, &BRes}
+	MultiMapToScalarField(scalars, []*Element{&A, &B})
```

Comment:
This situation is similar to many places in `go-verkle` where we have `Fr`s, and we would build the `[]*Fr` to get the results. 

cc @gballet